Policies/github-corporate-terms-of-service.md

```diff
@@ -65,12 +71,13 @@ If you would like to associate your Organization with a Corporate Account, we wi
 #### 5. Corporate Account Requirements
 You must create a User Account before you can create a Corporate Account. We have a few simple rules for individual Accounts on GitHub:
 
-- You must be a human to create an Account. Accounts registered by "bots" or other automated methods are not permitted. We do permit machine Account:
+- You must be a human to create an Account. Accounts registered by "bots" or other automated methods are not permitted. We do permit machine Accounts:
   - A machine account is an Account set up by an individual human who accepts the Terms on behalf of the Account, provides a valid email address, and is responsible for its actions. A machine account is used exclusively for performing automated tasks. Multiple users may direct the actions of a machine account, but the owner of the Account is ultimately responsible for the machine's actions. You may maintain no more than one free machine Account in addition to your free User Account.
 - One person or legal entity may maintain no more than one free Account (if you choose to control a machine account as well, that's fine, but it can only be used for running a machine).
 - You may not create an Account for the use of any User under the age of 13. While we are thrilled to see brilliant young coders get excited by learning to program, we must comply with United States law. GitHub does not target our Service to children under 13, and we do not permit any Users under 13 on our Service. If we learn of any User under the age of 13, we will [terminate that User’s Account immediately](#m-cancellation-and-termination). If you are a resident of a country outside the United States, your country’s minimum age may be older; in such a case, you are responsible for complying with your country’s laws.
 - Your login may only be used by one person — i.e., a single login may not be shared by multiple people. A paid organization Account may only provide access to as many User Accounts as its subscription allows.
 - Overall, the number of Users accessing Organizations associated with your Corporate Account must not exceed the number we have authorized for your Corporate Account.
+- You may not use GitHub in violation of export control or sanctions laws of the United States or any other applicable jurisdiction. You may not use GitHub if you are located in or ordinarily resident in a country or territory subject to comprehensive to comprehensive sanctions administered by the U.S. Office of Foreign Assets Control (OFAC). You may not use GitHub if you are or are working on behalf of a [Specially Designated National (SDN)](https://www.treasury.gov/resource-center/sanctions/SDN-List/Pages/default.aspx) or a person subject to similar blocking or denied party prohibitions. For more information, please see our [Export Controls policy(https://help.github.com/en/articles/github-and-export-controls).
```

Comment:
"to comprehensive to comprehensive" --- duplicate.

Reply:
```suggestion
- You may not use GitHub in violation of export control or sanctions laws of the United States or any other applicable jurisdiction. You may not use GitHub if you are located in or ordinarily resident in a country or territory subject to comprehensive sanctions administered by the U.S. Office of Foreign Assets Control (OFAC). You may not use GitHub if you are or are working on behalf of a [Specially Designated National (SDN)](https://www.treasury.gov/resource-center/sanctions/SDN-List/Pages/default.aspx) or a person subject to similar blocking or denied party prohibitions. For more information, please see our [Export Controls policy(https://help.github.com/en/articles/github-and-export-controls).
```
@nikitadanilov thanks for the feedback! Adding your suggested change above.